contracts/contracts/token/OUSD.sol

```diff
@@ -665,14 +665,33 @@ contract OUSD is Governable {
         );
         RebaseOptions stateFrom = rebaseState[from];
         RebaseOptions stateTo = rebaseState[to];
-        if (!_isNonRebasingAccount(from) &&
-                (stateFrom == RebaseOptions.NotSet ||
-                    stateFrom == RebaseOptions.StdRebasing)) {
+
+        require(
+            stateFrom == RebaseOptions.NotSet ||
+                stateFrom == RebaseOptions.StdNonRebasing ||
```

Comment:
so weird prettier does this tabulation for || expression but not for && expression below